games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit cmake-utils eutils flag-o-matic wxwidgets
+
+DESCRIPTION="lobby client for spring rts engine"
+HOMEPAGE="http://springlobby.info"
+SRC_URI="http://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="+translation debug +libnotify"
+
+RDEPEND="
```

Comment:
Please sort the dependencies by cat/pn. Unconditional go first, then all the conditionals sorted by flag name.

Reply:
Done